Extension/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.21.1: July 9, 2024
+### Enhancements
+* Add a `default` value for the `C_Cpp.experimentalFeatures` setting (enabling the `disabled` value to work). [#12163](https://github.com/microsoft/vscode-cpptools/issues/12163)
+* Add `see` and `sa` to the `C_Cpp.doxygen.sectionTags` setting. [#12384](https://github.com/microsoft/vscode-cpptools/issues/12384)
+* Update the vcpkg header database. [PR #12430](https://github.com/microsoft/vscode-cpptools/pull/12430)
+* Disable the pre-release prompt if the `extensions.ignoreRecommendations` setting is `true`. [#12438](https://github.com/microsoft/vscode-cpptools/issues/12438)
+* Switch to an alternative workspace symbol search implementation by default (unless `C_Cpp.experimentalFeatures` is `enabled`). [#12448](https://github.com/microsoft/vscode-cpptools/issues/12448)
+* Various IntelliSense engine updates/fixes.
+
+### Bug Fixes
+* Fix a crash if the Ryzen 3000 doesn't have updated drivers. [#12201](https://github.com/microsoft/vscode-cpptools/issues/12201)
+* Fix a tag parser performance regression (except when `C_Cpp.experimentalFeatures` is `enabled`). [#12292](https://github.com/microsoft/vscode-cpptools/issues/12292)
+* Fix code analysis, find all references, and rename from getting the wrong configuration for non-open files on the first run when using a configuration provider. [#12313](https://github.com/microsoft/vscode-cpptools/issues/12313)
+* Fix potential crashes. [#12354](https://github.com/microsoft/vscode-cpptools/issues/12354)
+* Fix the language status not showing it's busy while the tag parser is initializing. [#12403](https://github.com/microsoft/vscode-cpptools/issues/12403)
+* Fix the vcpkg code action not appearing for missing headers available via vcpkg. [#12413](https://github.com/microsoft/vscode-cpptools/issues/12413)
+* Fix custom configurations sometimes not getting used. [PR #12427](https://github.com/microsoft/vscode-cpptools/pull/12427)
+* Fix a code analysis error when using gcc 14. [#12428](https://github.com/microsoft/vscode-cpptools/issues/12428)
+* Fix colorization, inactive regions and inlay hints getting cleared when an update is pending.
```

Comment:
We might want to qualify this (and the line below about semantic tokens at the wrong location).  There are some other scenarios that have similar symptoms.  Maybe say "fixed a cause of"?

In the case of colorization getting cleared, there are scenarios where edge may not initially have all the info it needs to get colorization right for its 'first-block' update of the visible area, leading to colorization getting removed then added back in.

In the case of semantic tokens in the wrong places, I'm still seeing this occasionally and suspect it may be a symptom of: https://github.com/microsoft/vscode/issues/74094

Reply:
sure